egs/aidatatang_200zh/s5/local/format_data.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+
+if [ -f ./path.sh ]; then . ./path.sh; fi
```

Comment:
could be shortened as 
```
[ -f ./path.sh ] && . ./path.sh
```
but I think path.sh should always be present

Reply:
Yes, I have shortened it to ". ./path.sh".